src/app/_components/Footer/StacksSmiley.tsx

```diff
@@ -0,0 +1,34 @@
+import { useColorMode } from '@/components/ui/color-mode';
+import { StacksSmileyDarkIcon } from '@/ui/icons/StacksSmileyDarkIcon';
+import { StacksSmileyLaserEyesDarkIcon } from '@/ui/icons/StacksSmileyLaserEyesDarkIcon';
+import { StacksSmileyLaserEyesLightIcon } from '@/ui/icons/StacksSmileyLaserEyesLightIcon';
+import { StacksSmileyLightIcon } from '@/ui/icons/StacksSmileyLightIcon';
+import { Box, BoxProps, ClientOnly } from '@chakra-ui/react';
+import { useState } from 'react';
+
+export const StacksSmiley = (boxProps: BoxProps) => {
+  const { colorMode } = useColorMode();
+  const [isHovered, setIsHovered] = useState(false);
+  return (
+    <ClientOnly>
+      <Box
+        className="stacks-smiley"
+        onMouseEnter={() => setIsHovered(true)}
```

Comment:
I think we should try to achieve this with CSS so that the whole footer can be SSRed.